x-pack/plugins/security_solution/public/detection_engine/rule_exceptions_ui/pages/exceptions/exceptions_table.tsx

```diff
@@ -375,26 +402,60 @@ export const ExceptionListsTable = React.memo(() => {
           <Loader data-test-subj="loadingPanelAllRulesTable" overlay size="xl" />
         )}
 
-        {initLoading ? (
+        {initLoading || loadingTableInfo ? (
           <EuiLoadingContent data-test-subj="initialLoadingPanelAllRulesTable" lines={10} />
         ) : (
           <>
             <ExceptionsTableUtilityBar
               totalExceptionLists={exceptionListsWithRuleRefs.length}
               onRefresh={handleRefresh}
             />
-            <EuiBasicTable<ExceptionsTableItem>
-              data-test-subj="exceptions-table"
-              columns={exceptionsColumns}
-              isSelectable={hasPermissions}
-              itemId="id"
-              items={tableItems}
-              noItemsMessage={emptyPrompt}
-              onChange={handlePaginationChange}
-              pagination={paginationMemo}
-            />
+            <EuiSpacer size="m" />
+            {exceptionListsWithRuleRefs.length > 0 && (
+              <React.Fragment data-test-subj="exceptionsTable">
+                {exceptionListsWithRuleRefs.map((excList) => (
+                  <ExceptionsListCard
+                    key={excList.list_id}
+                    data-test-subj="exceptionsListCard"
+                    // @ts-expect-error canUserREAD and canUserCRUD are typed as boolean | null
+                    readOnly={canUserREAD && !canUserCRUD}
```

Comment:
`canUserREAD && canUserCRUD !== null && canUserCRUD === false;` can get rid of the ts-expect-error

Reply:
eh it was still complaining so I just moved the nullish checks to the conditional rendering logic a92fa177296